.github/ISSUE_TEMPLATE/package.yml

```
@@ -0,0 +1,63 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `package` label][package-issues] (including
+        closed issues) before you continue.
+
+
+        [package-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Apackage+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [Improvement, Bug, Other]
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Which packages do this issue concern?
```

This should be

```suggestion
      label: Which packages does this issue concern?
```

At least, I'm pretty sure, but English isn't my first language. If we can't agree, let try rephrasing :upside_down_face:

---

"do" is plural form the verb, "does" is for singular.

Example:
They do sports.
He does sports.

I don't think this should even be a question though, as none of the other labels are questions.

---

Perhaps "Relevant Packages" or "Related Packages"

---

> "do" is plural form the verb, "does" is for singular.
> 
> Example:
> They do sports.
> He does sports.

But the subject in this sentence is "the issue" not "packages", right?


> I don't think this label should even be a question though, as none of the other labels are questions.

That's a good point...



> Perhaps "Relevant Packages" or "Related Packages"

Don't quite like that, that sounds like it is about packages other than simple-icons.


What about following the current issue template and have "This issue concerns (the) ..." and have the options be "(The) NPM package" and "(The) Packagist package"? Alternatively, it could be "Ecosystem" or "Package manager".

---

> But the subject in this sentence is "the issue" not "packages", right?

You're totally right, sorry about that.

I like the "This issue concerns the..." solution. I would do it like this though: the label is "This issue concerns the..." and the options are "NPM package" and "Packagist package."

- [ ]

---

> You're totally right, sorry about that.

Cool, it's alright :+1: 


> I like the "This issue concerns the..." solution. I would do it like this though: the label is "This issue concerns the..." and the options are "NPM package" and "Packagist package."

Sounds good :+1: